### ISSUE TITLE
Number of matches requested

### ISSUE THREAD

**Issue by DrunkDutch** [andrewsnowden/dota2py#17]
Quick question regarding get_match_history, it seems to limit the matches_requested to 100, is there anyway to expand that to a higher limit or is it a fault on my part

**Comment by LukasSubel** [comment]
I would guess that limitation was included by Valve to prevent server overload.

**Comment by djmorrsee** [comment]
It's a limit of the API, the max is 100 matches returned.

A work around is to do multiple requests. Use the 'start_at_match_id' parameter with the id of the earliest match in the batch of 100 you get.

**Comment by DrunkDutch** [comment]
Yeah, I realized that, it's just that all the documentation on hand,
including the one linked too in the readme still has it listed as 25 games
per request.
